src/Notepads/Controls/TextEditor/TextEditorCore.cs

```diff
@@ -276,37 +285,51 @@ public void GetCurrentLineColumn(out int lineIndex, out int columnIndex, out int
 
             GetTextSelectionPosition(out var start, out var end);
 
-            lineIndex = 1;
-            columnIndex = 1;
+            startLineIndex = 1;
+            startColumnIndex = 1;
+            endLineIndex = 1;
+            endColumnIndex = 1;
             selectedCount = 0;
+            lineCount = _contentLinesCache.Length - 1;
```

Comment:
why -1 here?

Reply:
`_contentLinesCache = (_content + RichEditBoxDefaultLineEnding).Split(RichEditBoxDefaultLineEnding);`
Extra item introduced due to adding `RichEditBoxDefaultLineEnding `to `_content`.